cms/static/js/views/course_outline.js

```diff
@@ -209,6 +210,30 @@ define(["jquery", "underscore", "js/views/xblock_outline", "js/views/utils/view_
                         ensureChildrenRendered: this.ensureChildrenRendered.bind(this)
                     });
                 }
+            },
+
+            handleReIndexEvent: function(event) {
+                var self = this;
+                event.preventDefault();
+                var target = $(event.currentTarget);
+                target.css('cursor', 'wait');
+                this.startReIndex().done(function(data) {
+                    if(data.status !== 'success') {
```

Comment:
I assume that `this.startReIndex()` returns a [jQuery Deferred](http://api.jquery.com/category/deferred-object/) object. If so, `.done()` indicates that the object was resolved successfully -- you don't need to test `data.status == "success"`. If you want to handle the failure case, that's what `.fail()` handlers are for.